src/send/send_caliper_events.py

```diff
@@ -0,0 +1,64 @@
+import requests
+import json
+
+
+def api_handler(data):
```

Comment:
This should become a generic Python WAPI wrapper so that accessing the ESB is trivial.